evewspace/Map/models.py

```diff
@@ -597,6 +597,16 @@ def has_siblings(self):
             return False
         return parent_sys.childsystems.count() > 1
 
+    def distance_from_root(self):
```

Comment:
calculating this for each edit could be a performance issue with multiple concurrent users editing signatures